### ISSUE TITLE
Inquiry for Temporal Loss

### ISSUE THREAD

**Issue by jaisonoh** [AakashKT/pytorch-recurrent-ae-siggraph17#1]
Hi. I have a question for temporal loss

In your code, a temporal loss is implemented as below.
```python

def get_temporal_data(output, target):
	final_output = output.clone()
	final_target = target.clone()
	final_output.fill_(0)
	final_target.fill_(0)

	for i in range(1, 7):
		final_output[:, i, :, :, :] = output[:, i, :, :] + output[:, i-1, :, :]
		final_target[:, i, :, :, :] = target[:, i, :, :] + target[:, i-1, :, :]

	return final_output, final_target

def temporal_norm(output, target):
	return torch.sum(torch.abs(output - target)) / torch.numel(output)
```

You added previous and current frames for temporal output and target, respectively.
But in the paper, the authors mentioned that they used finite difference between them.

For reference, in the paper, 
"where a temporal derivative delta P_i / delta t for an i-th image pixel is computed
using finite differencing in time between the i-th pixels of the
current and the previous image in the temporal training sequence."

I think this is wrong. Can you check this?

Thanks.

**Comment by AakashKT** [comment]
Hi @jaisonoh ! Thank you for your query.
Apologies for replying late, it slipped out of my mind to address this issue earlier.

Yes, you are right, the paper does say finite differencing in time. Thanks for noticing, will update the code soon.
